packages/react-native-web/src/exports/Image/index.js

```diff
@@ -123,27 +126,50 @@ function resolveAssetUri(source): ?string {
           : prev
       );
     }
+
     const scaleSuffix = scale !== 1 ? `@${scale}x` : '';
-    uri = asset
-      ? `${asset.httpServerLocation}/${asset.name}${scaleSuffix}.${asset.type}`
-      : '';
+    const uri = `${asset.httpServerLocation}/${asset.name}${scaleSuffix}.${asset.type}`;
+    resolvedSource = { uri, width: asset.width, height: asset.height };
   } else if (typeof source === 'string') {
-    uri = source;
+    resolvedSource.uri = source;
+  } else if (Array.isArray(source)) {
+    if (process.env.NODE_ENV !== 'production') {
+      console.warn(
+        'The <Image> component does not support multiple sources passed as array, falling back to the first source in the list',
+        { source }
```

Comment:
```suggestion
        source
```
Are the brackets necessary?